external/llvm-project/mlir/test/Dialect/Tosa/tosa-partition-options.mlir

```diff
@@ -0,0 +1,72 @@
+// RUN: mlir-opt --tosa-partition %s | FileCheck %s
```

Comment:
A test file I happened to have laying around.  One batch of tests using command line, one using the API, same set of outputs.